overrides/home-content.mdx

```diff
@@ -0,0 +1,9 @@
+---
+title: Explore our changing planet.
+description: " "
+---
+
+import Landing from './home-content/landing.jsx';
+
+
+<Landing/>
```

Comment:
Heh okay, didn't know we are supposed to do it like this, i need to back out my changes here!

Reply:
No.. I don't think this file is actually necessary.. I am not sure how this file got into this -_-; I will get rid of this file, please keep your way.